brion/detail/meshHDF5.h

```diff
@@ -75,174 +78,166 @@ class MeshHDF5 : public Mesh
     MeshHDF5(const std::string& source, const bool overwrite,
              const MeshVersion version)
         : Mesh(source, version)
-        , _file()
     {
         try
         {
             SilenceHDF5 silence;
```

Comment:
I guess HDF5 is needed for this stuff. Ultimately this should also move to highfive